src/routes/router.js

```diff
@@ -85,7 +88,7 @@ export default Router.extend({
   faq() { this.render(<FAQ/>) },
   subscribe(id) { this.render(<FirstSubscriptionPage planName={id}/>) },
   userShow(id) { this.render(<UserShow userId={id}/>, {backgroundColor: 'GREY'}) },
-  organizationShow(id) { this.render(<OrganizationShow organizationId={id} key={id}/>, {backgroundColor: 'GREY'}) },
+    organizationShow(id) { this.render(<OrganizationShow organizationId={id} key={id} tab={extractOrganizationTab(window.location.search)}/>, {backgroundColor: 'GREY'}) },
```

Comment:
This is a pretty hacky way of doing this.  I believe there's a better way with redux router.  That said, it isn't that great either.  

In the least, I'd' hide 'window.location.search') within the function ``extractOrganizationTab``, which ideally would call a third function, ``extractSearchParams`` or similar.